Apps/WebClient/src/NewClientApp/src/components/TooManyRequestsComponent.vue

```diff
@@ -31,22 +31,22 @@ function clearTooManyRequestsError(): void {
 <template>
     <div>
         <v-alert
-            :show="showWarning"
+            :model-value="showWarning"
             data-testid="too-many-requests-warning"
             color="warning"
             dismissible
-            class="no-print"
+            class="no-print my-1"
```

Comment:
```suggestion
            class="d-print-none mb-4"
```
There is no `no-print` class.  Maybe it existed and was removed at one point, but it can be replaced with `d-print-none` for the same effect.

I also think these need a more significant margin, and just on the bottom, since they should almost always be positioned above other content.